src/client/containers/SessionContainer.tsx

```diff
@@ -76,6 +83,54 @@ export const setupSession = async (
   return sessionResult.data
 }
 
+type SetupQuoteCartSessionParams = {
+  quoteCartId: string
+  apolloClientUtils: ApolloClientUtils
+  storage: StorageState
+}
+
+export const setupQuoteCartSession = async ({
+  quoteCartId,
+  apolloClientUtils,
+  storage,
+}: SetupQuoteCartSessionParams): Promise<string> => {
+  if (!apolloClientUtils) {
+    throw new Error('Missing apollo client')
+  }
+
+  const accessTokenResult = await apolloClientUtils.client.mutate({
+    mutation: CREATE_ACCESS_TOKEN_MUTATION,
+    variables: { id: quoteCartId },
+  })
+
+  await afterTick(() => {
```

Comment:
Do we need this?

Reply:
Let's keep since it's copy from old code